crypto/keys/pem_signer_factory.go

```diff
@@ -72,10 +71,11 @@ func (f PEMSignerFactory) Generate(ctx context.Context, tree *trillian.Tree, spe
 		return nil, fmt.Errorf("Specification produces %T, but tree.SignatureAlgorithm = %v", key, tree.GetSignatureAlgorithm())
 	}
 
-	return marshalPrivateKey(key)
+	return MarshalPrivateKey(key)
 }
 
-func marshalPrivateKey(key crypto.Signer) (*any.Any, error) {
+// MarshalPrivateKey returns an any.Any with a keyspb.PrivateKey inside.
```

Comment:
I think this would be better placed at `private.go` now, considering that we've just made it public. I would also prefer for it to have its own set of tests.